### ISSUE TITLE
DFF_model Not Training

### ISSUE THREAD

**Issue by Korace0v0** [cake-lab/HybridDepth#15]
Hi, I find  `DFF_model` is not trained.  It is a little bit strange. 

And there is no `DFV_weights` provided for training.

And the following sentences also stop `DFF_model` from being trained.
```
        self.DFF_model.eval()
        self.DFF_model.requires_grad_(False)
```

The full code is in `model/depthNet.py`  as follows :

```
        self.DFF_model = DFFNet(clean=False, level=4, use_diff=1)
        self.DFF_model.cuda()
        DFV_weights = None  # Don't need this for evaluation, only add for training

        if DFV_weights is not None:
            pretrained_dict = torch.load(DFV_weights)
            pretrained_dict['state_dict'] = {
                k.replace('module.', ''): v
                for k, v in pretrained_dict['state_dict'].items()
                if 'disp' not in k
            }
            self.DFF_model.load_state_dict(
                pretrained_dict['state_dict'], strict=False)
            print('Loaded DFV model')
        else:
            print('Run with random init')

        self.DFF_model.eval()
        self.DFF_model.requires_grad_(False)
```

**Comment by AshkanGanj** [comment]
Hello, 
The weights that we provided includes the DFF model's weights as well, So you don't need to load it seperatly.  
If You want to train the DFF model you should remove it from eval mode and train the model end2end, which we never tried. Or if you want to reproduce our results you should train the DFF seperatly and plug it to the model and it will.

**Comment by AshkanGanj** [comment]
I close the issue, feel free to reopen it.
